src/StructureMap.DynamicInterception/MethodInvocation.cs

```diff
@@ -39,6 +54,21 @@ public MethodInfo MethodInfo
             get { return _invocation.Method; }
         }
 
+        public MethodInfo MethodInfoInvocationTarget
```

Comment:
`MethodInfo` points to the interface method while `InstanceMethodInfo` points to the implementation of the method. It is common to use attributes on the implementation of a method that need to be accessed by the interceptor. The attributes should not be put on the interface because they are implementation details.